build.gradle

```diff
@@ -7,11 +7,11 @@ plugins {
 }
 
 group 'org.hyperskill'
-version '12.0.3'
+version '13.0.0'
 
 java {
-    sourceCompatibility = JavaVersion.VERSION_17
-    targetCompatibility = JavaVersion.VERSION_17
+    sourceCompatibility = JavaVersion.VERSION_24
+    targetCompatibility = JavaVersion.VERSION_24
```

Comment:
Does this mean that users now need to install Java 24 separately to solve problems? After all, only Java 21 will come with the IDE

Reply:
Yes, it's true. Because all content will be updating to Java 24.